src/App.tsx

```diff
@@ -1,61 +1,137 @@
+import { FC, useState } from 'react';
+
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import { TodoList } from './components/TodoList';
+
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+
+const findUser = (param: number) => {
+  return usersFromServer
+    .find(user => user.id === param)
+    || null;
+};
+
+const preparedTodos = todosFromServer.map(todo => {
+  const user = findUser(todo.userId);
+
+  return {
+    ...todo,
+    user,
+  };
+});
+
+export const App: FC = () => {
+  const [todos, setTodos] = useState(preparedTodos);
+
+  const [title, setTitle] = useState('');
+  const [hasTitleError, setHasTitleError] = useState(false);
+
+  const [currentUserId, setCurrentUserId] = useState(0);
+  const [hasCurrentUserIdError, setHasCurrentUserIdError] = useState(false);
+
+  const maxId = todos.map((todo) => todo.id).sort((a, b) => b - a)[0];
```

Comment:
Instead of it you could use `reduce` method or` Math.max` to find a max id of the todo cause it more clear and readable